RESTserver/src/main/java/com/softserve/if072/restservice/service/StoreService.java

```diff
@@ -72,7 +78,7 @@ public Store getStoreByID(int id) throws DataNotFoundException {
      */
     @Transactional
     public void addStore(Store store) throws IllegalArgumentException {
-        if (store != null && store.getName() != null && store.getName() != "") {
+        if (store != null && store.getName() != null && !store.getName().equals("")) {
```

Comment:
can be replaced with 
`store != null && !"".equals(store.getName())`
and t is better to have literal to the left, because this prevent NPE